docs/installation.md

```diff
@@ -1,152 +1,182 @@
 # Installation
 
-##  Installing and running the server and database containers 
-The quipucords container image can be created from source. This quipucords repository includes a Dockerfile that contains instructions for the image creation of the server.
-You must have [Docker installed](https://docs.docker.com/engine/installation/) or [Podman installed](https://podman.io/getting-started/installation).
-The examples below all use `podman` but can be replaced with `docker` unless stated otherwise.
-1. Clone the repository:
+This document describes two methods of building and running quipucords containers locally from source for testing and development:
+
+1. install and run containers in a podman pod
+2. install and run containers via podman compose
+
+These methods are not officially supported for downstream users. Downstream users should always refer to official Red Hat documentation for running the Red Hat-managed downstream container images.
+
+## Podman Setup
+
+See also [Podman installation docs](https://podman.io/getting-started/installation).
+
+* If running on Fedora/Centos/RHEL,
    ```
-   git clone git@github.com:quipucords/quipucords.git
+   sudo subscription-manager register
```

Comment:
The `sudo subscription-manager register` instruction was part of the original instructions, and it only appears in the diff because I rearranged some lines. I'll leave it here since it doesn't do any harm, and it might be useful in case someone has customized their system in a way that this might be required.